features/property_address.feature

```diff
@@ -0,0 +1,31 @@
+@add_property_address
+  Feature: Conveyancer provides a single property address
+  As a conveyancer I want to be able to provide a property address
+  So that it can be included on the mortgage deed
+
+
+Acceptance Criteria
+  -Property provided must be a single property address
+  -It needs to look like an address
+  -Commas provided in the address indicate a new line (apart from when it follows
+   a number or a number followed by a letter in the first)
+  -Commas are not displayed on the address when borrower views deed
```

Comment:
Really minor but I started adding the user story number e.g. (US106) before each of the acceptance criteria just to help identify which scenarios potentially cover which criteria now me have many us scenarios in a feature file.